addons/hr_holidays/models/hr_accrual_plan.py

```diff
@@ -0,0 +1,90 @@
+# # -*- coding: utf-8 -*-
+
+import datetime
+import logging
+
+from odoo import api, fields, models
+from odoo.exceptions import ValidationError
+from odoo.tools.translate import _
+
+_logger = logging.getLogger(__name__)
+
+class AccrualPlan(models.Model):
+    _name = "hr.accrual.plan"
+    _description = "Accrual Plan"
+    _order = 'sequence, id'
+
+    name = fields.Char('Accrual Plan', required=True)
+    company_id = fields.Many2one('res.company', string='Company', required=True, readonly=True,default=lambda self: self.env.company)
+    sequence = fields.Integer('Level', default=1)
+    start_after_count = fields.Float(required=True,default=0)
+    start_after_type = fields.Selection([('days', 'days'),('months', 'months'), ('years', 'years')],default='days')
+
+    added_hours = fields.Float(required=True)
+    maximum_hours = fields.Float('Maximum accrual hours')
+   
+    frequency = fields.Selection([
+        ('daily','daily'),
+        ('weekly','weekly'),
+        ('every other week','every other week'),
+        ('twice a month','twice a month'),
+        ('monthly','monthly'),
+        ('twice a year','twice a year'),
+        ('quarterly','quarterly'),
+        ('yearly','yearly'),
+        ('anniversary','anniversary'),
+        ('per hours worked','per hour worked')
+        ], default='per hours worked')
+
+    # pivot_ids=fields.One2many('hr.accrual.pivot', 'plan_id')
+
+    period_weekday = fields.Selection('_weekdays', default='1')
```

Comment:
Following the guidelines
```suggestion
    period_weekday = fields.Selection(selection='_get_period_weekdays_selection', default='1')
```